ts/navigation/OnboardingNavigator.tsx

```diff
@@ -49,6 +50,10 @@ const navigator = () => (
       name={ROUTES.ONBOARDING_FINGERPRINT}
       component={FingerprintScreen}
     />
+    <Stack.Screen
+      name={ROUTES.NEW_ONBOARDING_INSERT_EMAIL_SCREEN}
+      component={NewOnboardingEmailInsertScreen}
+    />
```

Comment:
Since this screen will replace the old one, I'd not add its navigation management. What do you think? If we want to test it we could just replace the component name in the `ROUTES.ONBOARDING_INSERT_EMAIL_SCREEN`. What do you think?

Reply:
fixed!